mona_sdk/client.py

```diff
@@ -763,13 +785,21 @@ def validate_config(
         A wrapper function for "Validate Config" REST endpoint. view full documentation
         here: https://docs.monalabs.io/docs/validate-config-via-rest-api
         """
-        return self._app_server_request(
+        app_server_response = self._app_server_request(
             "validate_config",
             data={
                 "config": config,
                 "list_of_context_ids": list_of_context_ids,
                 "latest_amount": latest_amount,
             },
+        )["response_data"]
+
+        return (
+            self._handle_service_error(
+                (list(app_server_response["issues"].values())[0]).__str__()
```

Comment:
what is this __str__ thing? this isn't the way to cast to str - you should do f"{obj}" to turn it into string.
also what if issues is empty? you expect it to have [0] - seems unsafe